### ISSUE TITLE
Keyboard shortcut not working - Is it only me?

### ISSUE THREAD

**Issue by sarbbottam** [dkundel/vscode-new-file#5]
I could see the shortcut ⌥⌘N specified against the command.
![screen shot 2016-02-27 at 12 50 09 am](https://cloud.githubusercontent.com/assets/949380/13372065/2067b990-dcec-11e5-8593-79e1dcc2f232.png)

However, could not get the prompt when I press ⌥⌘N. 
Could you please verify?

Thanks!!

**Comment by skol-pro** [comment]
Same here, using Mac OS too.

**Comment by Treri** [comment]
same here.
using Code 0.10.11 and OS X 10.11.2

**Comment by hackerone** [comment]
same here using - Version 0.10.11 (0.10.11)

**Comment by chuckhendo** [comment]
It appears that VSCode interprets the "N" as if you pressed "Opt+N" (special character). So, if you change your keybinding to the following, it should work:

{
    "key": "alt+cmd+`",
    "command": "extension.createNewFile"
}

**Comment by sarbbottam** [comment]
>It appears that VSCode interprets the "N" as if you pressed "Opt+N" 

Sorry, I don't follow you, shortcut already suggests ⌥⌘N. I am pressing ⌥N (Opt+N) along with ⌘ (Cmd), Why VSCode is confused? Do you mean to say VSCode is considering ⌥⌘N as ⌥⌘⌥N.

Thats strange. There are quite a few commands that uses ⌥⌘ + Letter as shortcut, please refer https://code.visualstudio.com/Docs/customization/keybindings, a quick search for ⌥⌘, would highlight the result.

**Comment by chuckhendo** [comment]
I'm honestly not sure, but the way I discovered this was by hitting "CMD+K CMD+K" in the keybinding screen, and then trying ⌥⌘N. It entered it into VSCode as "alt+cmd+`"

**Comment by Treri** [comment]
@chuckhendo I tried. Yes, if I press ⌥⌘N. It entered it into VSCode as "alt+cmd+`"

Maybe we should report a issure to vscode

**Comment by dkundel** [comment]
Thanks for the investigation folks! I'll create an issue in the vscode repository. 

Since this seems to be a Mac specific bug I won't publish a new update for this but I verified the suggestion from @chuckhendo and it indeed works. So feel free to throw that into your keybindings as a temporary workaround:

```json
{
  "key": "alt+cmd+`",
  "command": "extension.createNewFile"
}
```

**Comment by dkundel** [comment]
The issue is now open under microsoft/vscode#5164 in case you are interested!

**Comment by dkundel** [comment]
I pushed a new version so you should be able to update it to version 0.1.0!

**Comment by sarbbottam** [comment]
W00t!  

BTW, the latest version is `0.1.1` not `0.1.0`

**Comment by dkundel** [comment]
Oh yeah that's because I realized afterwards that I had forgot something :smile:

**Comment by etiennejcharles** [comment]
I believe this should be re-opened ( @dkundel Please validate ) 

On Mac OS : 10.12.4
VS Code : 1.11.1 

In order to make it work. 
I had to change it back to **cmd+alt+n** in my key bindings.

I believe the default is: **alt+cmd+`** to circumvent the original problem discussed here, which seems ~ to have been fixed in a newer version of vscode.

**Comment by dkundel** [comment]
You are correct @etiennejcharles!

They fixed the actual VS Code problem and therefore this will have to be reverted. I'll push out a new update this weekend.

Thanks for raising this!

Dominik

**Comment by mitchellporter** [comment]
Any update on this? Still experiencing this behavior.

**Comment by dkundel** [comment]
Hey @mitchellporter, @etiennejcharles  I finally just shipped an update to v3.0.0 that should fix this again and ship a bunch of new things that you can find in the change log.

Please let me know if you encounter any other problems.

Cheers,
Dominik
